pvactools/tools/pvacseq/run.py

```diff
@@ -74,6 +74,14 @@ def main(args_input = sys.argv[1:]):
     input_file_type = 'vcf'
     base_output_dir = os.path.abspath(args.output_dir)
 
+    if (args.netmhc_pan_version == '4.0' and args.iedb_install_directory is not None):
+        raise Exception("Standalone IEDB does not support version 4.0")
+
+    if (args.netmhc_pan_version and ("NetMHCIIpan" in args.prediction_algorithms or "NetMHCIIpanEL" in args.prediction_algorithms)):
+        NetMHCIIVersion.netmhcii_pan_version = args.netmhc_pan_version
+    elif (args.netmhc_pan_version and not ("NetMHCIIpan" in args.prediction_algorithms or "NetMHCIIpanEL" in args.prediction_algorithms or "all_class_ii" in args.prediction_algorithms)):
+        raise Exception("NetMHCIIpan and NetMHCIIpanEL version was specified but neither algorithm was selected")
```

Comment:
I think we can silently ignore this and don't need to error out. There is no adverse effect if it is set but not used. Additionally, this check would now always error when we add a default for the version, as suggested.